webfront/views/cache.py

```diff
@@ -49,14 +70,6 @@ def get_timeout_from_path(path, endpoint_levels):
                 return FIVE_DAYS
         except Exception:
             return SHOULD_NO_CACHE
-    short_life_parameters = [
-        "cursor",
-        "size",
-        "go_terms",
-        "ida_ignore",
-        "ida_search",
-        "format",
-    ]
     for parameter in short_life_parameters:
         value = query.get(parameter)
         if value is not None:
```

Comment:
```suggestion
        if parameter is query:
```

Reply:
I don't get this suggestion

Reply:
Yeah... I should have added a comment. :man_facepalming: 
It is just at attempt to simplify the code: the logic to check whether the query has one of the short life parameters doesn't require the value associated to the param, so might as well just check for the presence of `parameter` in `query`.

Reply:
Yes... (I need a double-facepalm emoji :confounded:)